## H3ac: Acetylation of histone H3

<http://en.wikipedia.org/wiki/Histone_H3>:
The N-terminal tail of histone H3 protrudes from the globular nucleosome core and can undergo several different types of post-translational modification that influence cellular processes. These modifications include the covalent attachment of methyl or acetyl groups to lysine and arginine amino acids and the phosphorylation of serine or threonine. Di- and Tri-methylation of Lysine 9 are associated with repression and heterochromatin, while mono-methylation of K4 is associated with active genes.[3] has been associated with gene silencing and the formation of relatively inactive regions of DNA known as heterochromatin.[4] Acetylation of histone H3 occurs at several different lysine positions in the histone tail and is performed by a family of enzymes known as histone acetyltransferases (HATs). Acetylation of lysine14 is commonly seen in genes that are being actively transcribed into RNA.


## CTCF: negative regulator

<http://en.wikipedia.org/wiki/CTCF>

The primary role of CTCF is thought to be in regulating the 3D structure of chromatin.[4] CTCF binds together strands of DNA, thus forming chromatin loops, and anchors DNA to cellular structures like the nuclear lamina.[6] It also defines the boundaries between active and heterochromatic DNA.

Since the 3D structure of DNA influences the regulation of genes, CTCF's activity influences the expression of genes. CTCF is thought to be a primary part of the activity of insulators, sequences that block the interaction between enhancers and promoters. CTCF binding has also been both shown to promote and repress gene expression. It is unknown whether CTCF affects gene expression solely through its looping activity, or if it has some other, unknown, activity.[4]


## RAD21: Double-strand-break repair protein

<http://en.wikipedia.org/wiki/RAD21>



